src/jdk.hotspot.agent/doc/transported_core.html

```diff
@@ -76,16 +76,35 @@ <h3>Using transported core dumps on Windows</h3>
  is setup:
  </p>
 
-<p>
+<code>
  set PATH=%JAVA_HOME%\bin;%JAVA_HOME%\bin\server;%PATH%
-</p>
+</code>
 
 <p>
 You can also include user JNI libraries in <b>PATH</b>.
 If the windows libraries are not identical, then they may also need to be copied
 to the debugger machine and included in <b>PATH</b>.
 </p>
 
+<p>
+By default symbols are also located using <b>PATH</b>. However, there are also Java properties that can be used to specify both the location of the binaries, and also separately the location of symbols. Use <b>sun.jvm.hotspot.debugger.windbg.imagePath</b> for the location of binaries, and use <b>sun.jvm.hotspot.debugger.windbg.symbolPath</b> for the location of symbols. <b>imagePath</b> defaults to <b>PATH</b> if not set, and <b>symbolPath</b> defaults to <b>imagePath</b>. The advantage of using these propeties is that you don't need to change your <b>PATH</b> setting, and they allow for binaries to be located separately from symbols.
+</p>
+
+<p>
+How you set these properties will depend on the SA tool being used. The following in an example of when launching the clhsdb tool:
+</p>
+
+<code>
+jhsdb -J-Dsun.jvm.hotspot.debugger.windbg.imagePath="%PATH%;D:/SomePath" clhsdb
+</code>
+
+<p>If you are not seeing symbols for Windows libraries, try adding "<b>srv*https://msdl.microsoft.com/download/symbols</b>" to <b>sun.jvm.hotspot.debugger.windbg.symbolPath</b>. Note doing this means that SA will no longer look in <b>PATH</b> for your JVM and JNI symbols, so make sure those paths also get added. The following is one possible approach:
+</p>
+
+<code>
+jhsdb -J-Dsun.jvm.hotspot.debugger.windbg.symbolPath="%PATH%;srv*https://msdl.microsoft.com/download/symbol" clhsdb
+</code>
+
```

Comment:
If you are not seeing symbols for Windows libraries, try setting sun.jvm.hotspot.debugger.windbg.symbolPath to include "srv*https://msdl.microsoft.com/download/symbols". Usually also include PATH, for example:

jhsdb -J-Dsun.jvm.hotspot.debugger.windbg.symbolPath="%PATH%;srv*https://msdl.microsoft.com/download/symbol" clhsdb

Reply:
Just seeing if that could be clearer.  Whichever you like is fine. 8-)

Reply:
fixed